lib/paperclip/io_adapters/nil_adapter.rb

```diff
@@ -1,6 +1,6 @@
 module Paperclip
   class NilAdapter < AbstractAdapter
-    def initialize(target)
+    def initialize(target, options = {})
```

Comment:
Unused method argument - `target`. If it's necessary, use `_` or `_target` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `initialize(*)` if you want the method to accept any arguments but don't care about them.